gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjurePlugin.java

```diff
@@ -270,27 +271,38 @@ private static boolean shouldConfigureJavaServices(Project project) {
         return !(projectName.endsWith(JAVA_OBJECTS_SUFFIX) || projectName.endsWith(JAVA_UNDERTOW_SUFFIX));
     }
 
-    private static void setupObjectsProject(Project project) {
+    private static void setupObjectsProject(Project project, Supplier<GeneratorOptions> _optionsSupplier) {
         project.getDependencies().add("api", Dependencies.CONJURE_JAVA_LIB);
         project.getDependencies().add("implementation", Dependencies.JETBRAINS_ANNOTATIONS);
     }
 
-    private static void setupDialogueProject(Project project) {
+    private static void setupDialogueProject(Project project, Supplier<GeneratorOptions> _optionsSupplier) {
         project.getDependencies().add("api", Dependencies.DIALOGUE_TARGET);
     }
 
-    private static void setupRetrofitProject(Project project) {
+    private static void setupRetrofitProject(Project project, Supplier<GeneratorOptions> optionsSupplier) {
+        boolean useJakarta = optionsSupplier.get().has("useJakartaNamespaces")
+                && ((Boolean) optionsSupplier.get().get("useJakartaNamespaces"));
         project.getDependencies().add("api", "com.google.guava:guava");
         project.getDependencies().add("api", "com.squareup.retrofit2:retrofit");
-        project.getDependencies().add("compileOnly", Dependencies.ANNOTATION_API);
+        project.getDependencies()
+                .add(
+                        "compileOnly",
+                        useJakarta ? Dependencies.ANNOTATION_API_JAKARTA : Dependencies.ANNOTATION_API_JAVAX);
     }
 
-    private static void setupJerseyProject(Project project) {
-        project.getDependencies().add("api", Dependencies.JAXRS_API);
-        project.getDependencies().add("compileOnly", Dependencies.ANNOTATION_API);
+    private static void setupJerseyProject(Project project, Supplier<GeneratorOptions> optionsSupplier) {
+        boolean useJakarta = optionsSupplier.get().has("useJakartaNamespaces")
+                && ((Boolean) optionsSupplier.get().get("useJakartaNamespaces"));
```

Comment:
let's either extract a constant, or assuming this returns null when a value  doesn't exist, something like `Boolean.TRUE.equals(optionsSupplier.get().get("useJakartaNamespaces"))`

Reply:
`.get(...)` throws when a value doesn't exist :(

Reply:
how do you feel about https://github.com/palantir/gradle-conjure/pull/1155/commits/427fd95357866750caea9ad0b916075d09bf0a58#diff-57a6e01c74489ed759713fa386614f0d7bd705d8cba6c7dfaa7f27f8baa9a4e9R38-R43?

Reply:
lgtm